src/algolia/utils.js

```diff
@@ -0,0 +1,48 @@
+// Mapping between display categories and their Algolia index property path
+// Used for populating the Dataset Search Results facet menu dynamically
+export const facetPropPathMapping = {
+  'anatomy.organ.name' : 'Anatomical Structure',
+  'organisms.primary.species.name' : 'Species',
+  'item.modalities.keyword' : 'Experimental Approach',
+  'attributes.subject.sex.value' : 'Sex',
+  'attributes.subject.ageCategory.value' : 'Age Categories',
+}
+
+export const getAlgoliaFacets = function(algoliaIndex, propPathMapping) {
+  const map = new Map(Object.entries(propPathMapping));
+  const facetPropPaths = Array.from(map.keys() );
+  var facetData = []
```

Comment:
I found multiple occurrences of vars and lets. It is better and safer to use let to avoid problem with scopes.

Reply:
Ah yeah good catch! will fix